ui/apps/dashboard/src/components/Pill/HorizontalPillList.tsx

```diff
@@ -20,24 +20,19 @@ export default function HorizontalPillList({
     return (
       <>
         {alwaysVisiblePills}
-        <Tooltip.Provider>
-          <Tooltip.Root delayDuration={0}>
-            <Tooltip.Trigger className="cursor-default">
-              <Pill className="bg-white px-2.5 align-middle text-slate-600">
-                +{hiddenPills.length}
-              </Pill>
-            </Tooltip.Trigger>
-            <Tooltip.Portal>
-              <Tooltip.Content
-                className="data-[state=delayed-open]:data-[side=top]:animate-slide-down-and-fade data-[state=delayed-open]:data-[side=right]:animate-slide-left-and-fade data-[state=delayed-open]:data-[side=left]:animate-slide-right-and-fade data-[state=delayed-open]:data-[side=bottom]:animate-slide-up-and-fade text-violet11 select-none rounded-[4px] bg-white px-[15px] py-[10px] text-[15px] leading-none shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] will-change-[transform,opacity]"
-                sideOffset={5}
-              >
-                <div className="flex flex-col gap-2">{hiddenPills}</div>
-                <Tooltip.Arrow className="fill-white" />
-              </Tooltip.Content>
-            </Tooltip.Portal>
-          </Tooltip.Root>
-        </Tooltip.Provider>
+        <Tooltip delayDuration={0}>
+          <TooltipTrigger className="cursor-default">
+            <Pill className="bg-white px-2.5 align-middle text-slate-600">
+              +{hiddenPills.length}
+            </Pill>
+          </TooltipTrigger>
+          <TooltipContent
+            className="data-[state=delayed-open]:data-[side=top]:animate-slide-down-and-fade data-[state=delayed-open]:data-[side=right]:animate-slide-left-and-fade data-[state=delayed-open]:data-[side=left]:animate-slide-right-and-fade data-[state=delayed-open]:data-[side=bottom]:animate-slide-up-and-fade text-violet11 select-none rounded-[4px] bg-white px-[15px] py-[10px] text-[15px] leading-none shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] will-change-[transform,opacity]"
```

Comment:
I need to test but I'm pretty sure we can remove the styles from the TooltipContent of all items we are replacing, as the shared component already has styles.